.github/workflows/deploy.yml

```diff
@@ -0,0 +1,41 @@
+name: Deploy to Crates.io
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseType:
+        description: 'Release type (major, minor, patch)'
+        required: true
+        default: 'patch'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+      - name: Cargo bump
+        run: |
+          cargo install cargo-bump
+          cargo bump ${{ github.event.inputs.releaseType }}
+      - name: Retrieve new version
+        run: |
+          echo "::set-output name=CARGO_VERSION::$(grep -e '^version' Cargo.toml | cut -d "=" -f2 | tr -d '/"' | tr -d ' ')"
+        id: version
+      - name: Build one time, for sanity
+        run: cargo build
+      - uses: katyo/publish-crates@v1
```

Comment:
I did a quick visual audit of their code and didn't see any security concerns with this having access to the crates.io credentials, but we should probably give it a closer look before handing credentials to it.

Reply:
I did a more thorough audit and I feel that we can trust the `v1` version with the token. It appears to pull it in and immediately hand it off to `cargo publish`.